iroh-blobs/src/util/local_pool.rs

```diff
@@ -568,6 +572,7 @@ mod tests {
         tokio::time::sleep(Duration::from_millis(100)).await;
         // drop x at the end. we will never get here when the future is
         // no longer polled, but drop should still be called
+        tracing::debug!("Dropping {x:?}");
```

Comment:
@dignifiedquire should this line still be caught in logging? It's not currently.